Packs/PAN-OS/Integrations/Panorama/README.md

```diff
@@ -6185,17 +6185,18 @@ Get all the jobs from the devices in the environment, or a single job when ID is
 | **Path** | **Type** | **Description** |
 | --- | --- | --- |
 | PANOS.JobStatus.hostid | String | Host ID. | 
-| PANOS.JobStatus.id | String | ID of job. | 
+| PANOS.JobStatus.id | String | ID of job. For failed jobs, the ID will be -1. | 
 | PANOS.JobStatus.type | String | Job type. | 
 | PANOS.JobStatus.tfin | String | Time finished. | 
 | PANOS.JobStatus.status | String | Status of the job. | 
 | PANOS.JobStatus.result | String | The result of the job. | 
-| PANOS.JobStatus.user | String | The user who initiated the job. | 
+| PANOS.JobStatus.user | Unknown | The user who initiated the job. | 
```

Comment:
why this change?

Reply:
In most cases the value returned is `null`